packages/components/src/components/primitives/form/DropdownField/DropdownDownshiftSelect/DropdownDownshiftSelect.tsx

```diff
@@ -48,6 +48,10 @@ export const DropdownDownshiftSelect = forwardRef<
       setItemSelected(selectedItem?.value);
     };
 
+    useEffect(() => {
+      initialSelectedOption && handleOnChange(initialSelectedOption);
+    }, []);
```

Comment:
this is awful, do we really need it?

Reply:
Yep, Downshift doesn't trigger `onItemChange` when `initialSelectedOption` is passed so we can't validate this dropdown

Reply:
of course it does not trigger the event because the data has not changed, same reason why I do not like this change at all